site/content/contribute/mobile/push-notifications/corporate-proxy.md

```diff
@@ -0,0 +1,37 @@
+---
+title: "Corporate Proxy Server"
+date: 2020-03-09T11:35:32
+weight: 3
+subsection: "Setup Push Notifications"
+---
+
+### How do I receive mobile push notification if my IT policy requires the use of a corporate proxy server?
+
+When your IT policy requires a corporate proxy to scan and audit all outbound traffic the following options are available:
+
+###### 1 - Deploy Mattermost with connection restricted post-proxy relay in DMZ or a trusted cloud environment
+
+Some legacy corporate proxy configurations may be incompatible with the requirements of modern mobile architectures, such as the requirement of HTTP/2 requests from Apple to send push notifications to iOS devices.
+
+In this case, a [post-proxy relay](https://docs.mattermost.com/overview/faq.html#what-are-pre-proxy-and-post-proxy-relays) can be deployed to take messages from the Mattermost server passing through your corporate IT proxy in the incompatible format, e.g. HTTP/1.1, transform it to HTTP/2 and relay it to its final destination, either to the [Apple Push Notification Service (APNS)](https://developer.apple.com/library/archive/documentation/NetworkingInternet/Conceptual/RemoteNotificationsPG/APNSOverview.html#//apple_ref/doc/uid/TP40008194-CH8-SW1) and [Google Fire Cloud Messaging (FCM)](https://firebase.google.com/docs/cloud-messaging) services. 
+
+Ths **post-proxy relay** [can be configured using the Mattermost Push Proxy installation guide](https://developers.mattermost.com/contribute/mobile/push-notifications/service/) with connection restrictions to meet your custom security and compliance requirements.
+
+In place of a DMZ you can also host in a trusted cloud environment such as AWS or Azure depending on your internal approvals and policies. 
+
+![image](/img/mobile/post-proxy-relay.png)
```

Comment:
Image not rendering in preview mode, but I think that's just due to the nature of the GitHub preview rendering. Should work in developers.mattermost.com once pushed live.

Reply:
How should I proceed from here?

Reply:
We can leave it unchanged in this PR, and check the image renders once it's live in developers.mattermost.com